components/brave_ads/common/pref_names.cc

```diff
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/components/brave_ads/common/pref_names.h"  // IWYU pragma: keep
+#include "brave/components/brave_ads/common/pref_names.h"  // IWYU pragma: export
```

Comment:
As I understand, `// IWYU pragma: export` has meaning only in `"*.h"` files. Should we keep `// IWYU pragma: keep` here?

Reply:
Great catch, thanks